### PR TITLE
 Setup server in the Rust app #55 

### PR DESCRIPTION
**Pull request by Blindspot22**
### Description:

This task focuses on enhancing our Rust project by integrating a base server using the Rocket framework and adding support for serialization and deserialization with Serde, serde_json, and serde_derive. The goal is to lay a solid foundation for our application's backend, allowing for efficient data handling and a robust server setup.
Objectives:

**Add Dependencies**: Include Rocket, serde, serde_json, and serde_derive in the project's Cargo.toml.

**Configure Rocket**: Set up a basic Rocket server to handle incoming HTTP requests.

**Implement Serde**: Utilize Serde for serializing and deserializing data structures, ensuring seamless data exchange between the server and clients.

**Basic Endpoint Creation**: Create a simple API endpoint to demonstrate the use of Rocket and Serde together.
    Update Dockerfile: The port of the server should be exposed on the Dockerfile.

### Acceptance Criteria:

The Cargo.toml file is updated with the latest versions of Rocket, serde, serde_json, and serde_derive.
A Rocket server is initialized and runs successfully, listening for incoming HTTP requests on a specified port.
At least one API endpoint is created, using Serde to serialize and deserialize request and response data.
The project compiles without errors, and the endpoint is accessible and correctly handles JSON data as per Serde configurations.
Documentation is provided on how to start the server and interact with the created endpoint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ Before getting started, make sure you have the following tools installed:
 
 - [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm): To manage Node.js versions.
 - [Node.js](https://nodejs.org/): Javascript runtime environment.
-- [npm](https://www.npmjs.com/): Package manager for Node.js.
-- [sdkman](https://sdkman.io/): Software Development Kit Manager.
```
Comment (stephane-segning) [inline README.md:11]: Why are you removing this?

**Review by stephane-segning — COMMENTED** [review]
Awesome work! Can you help me understand these changes?

File: README.md (hunk 1 above)
Comment (Blindspot22) [reply README.md:11]: @stephane-segning  Surely a mistake from @Tekum-Emmanuella during the documentation. We began by including it in the README file then later on decided to create our own doc

Comment (stephane-segning) [reply README.md:11]: Ok.

Comment (stephane-segning) [reply README.md:11]: It should be fixed then

**Comment by stephane-segning** [comment]
The github action is failing. Please @Blindspot22 @Tekum-Emmanuella @ndefokou @Calebasah  fix this

**Comment by Blindspot22** [comment]
> The github action is failing. Please @Blindspot22 @Tekum-Emmanuella @ndefokou @Calebasah fix this

Okay sir @stephane-segning

File: power-pay-translator/src/main
Comment (stephane-segning) [inline power-pay-translator/src/main:1]: Why is there a binary file here?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Review by stephane-segning — APPROVED** [review]
Awesome!

File: power-pay-translator/src/main
Comment (stephane-segning) [inline power-pay-translator/src/main:1]: Now remove this binary from here

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: power-pay-translator/src/main
Comment (Blindspot22) [reply power-pay-translator/src/main:1]: > Now remove this binary from here

Ohhh so sorry @stephane-segning it's today i saw your message. But rest assured it has been done you can verify.

**Review by stephane-segning — APPROVED** [review]
Nice work

**Pull request merged**